target/linux/pistachio/creator-platform-cascoda-default.config

```diff
@@ -11,4 +11,3 @@ CONFIG_VERSION_REPO="http://downloads.creatordev.io/pistachio/marduk/packages"
 CONFIG_VERSION_MANUFACTURER="Imagination Technologies"
 CONFIG_VERSION_MANUFACTURER_URL="www.imgtec.com"
 CONFIG_VERSION_PRODUCT="Creator Ci40(Marduk)"
-CONFIG_VERSION_HWREV="Rev5 with CA8210"
```

Comment:
I was rather suggesting to remove "Rev" word, but keep HWREV as say "Ci40 with CA8210"